libevmasm/Instruction.cpp

```diff
@@ -219,6 +220,7 @@ static std::map<Instruction, InstructionInfo> const c_instructionInfo =
 	{Instruction::SHL,            {"SHL",             0,  2,   1,  false,      Tier::VeryLow}},
 	{Instruction::SHR,            {"SHR",             0,  2,   1,  false,      Tier::VeryLow}},
 	{Instruction::SAR,            {"SAR",             0,  2,   1,  false,      Tier::VeryLow}},
+	{Instruction::CLZ,            {"CLZ",             0,  1,   1,  false,      Tier::Low}},
```

Comment:
`CLZ` has the same gas cost as `ADD`, which is in the `Tier::VeryLow`.

Reply:
I think it was updated. Also it is `5` in geth. Plus in the EIP it mentions:

> The cost of the opcode is 5, matching MUL (raised from 3 to avoid under-pricing DoS risk).

Reply:
this is fine